usage.go

```diff
@@ -33,6 +33,12 @@ path:
 params:
   ?freq=        Sets QSY frequency (ardop and ax25 only)
   ?host=        Overrides the host part of the path. Useful for serial-tnc to specify e.g. /dev/ttyS0.
+  ?prehook=     Sets an executable middleware to run before the connection is handed over to the B2F protocol.
```

Comment:
High-level question: since we are making a connection pre-hook for initializing connections, does that now require that there's a post-hook for tearing the connection down cleanly? Maybe the proposed use case, AX.25 packet, is smart enough to tear down traversal links.

Reply:
That's a very good question! 🤔 

I would imagine that all connections would be teared down once one of the links are disconnected. Otherwise, you'd run the risk of dangling connections in case the dialer looses it's connection. But this is definitely something to investigate.

Reply:
We could check to see if Winlink Express provides such a feature. If they don't, I would imagine it's not very common 😁